third-party/realdds/include/realdds/dds-stream-server.h

```diff
@@ -22,10 +22,11 @@ class dds_stream_profile;
 struct image_header
 {
     int format;
-    int height = 0;
-    int width = 0;
+    int height = -1;

```

Comment:
Why? Do we have a case where 0x0 is valid?

Reply:
For motion streams width and height are 0